### ISSUE TITLE
podman farm build creates directories and files under cwd

### ISSUE THREAD

**Issue by edsantiago** [containers/podman#20505]
As a general principle, tools should not unexpectedly mess around with their working directory.

Perhaps this is a known issue, maybe it was kludged that way as a stopgap, but this must be fixed before releasing into the wild.

**Comment by umohnani8** [comment]
So the farm build work is being redesigned to push directly to the registry. We merged the first PR in to get most of the setup code in so that the future rework PR is smaller. This command is currently hidden and the current code will not be used once the redesigned code lands in. 
@edsantiago I can open a PR to add a comment that this will be removed in a few weeks once the new design is in.

**Comment by rhatdan** [comment]
In the final design `podman-remote farm build` will not be touching the homedir.

**Comment by edsantiago** [comment]
>open a PR to add a comment that this will be removed in a few weeks

Not worth the bother. Just keep this issue open so it doesn't get lost. Thank you!

**Comment by afbjorklund** [comment]
Writing to the current directory (in 4.8.3) was actually easier to understand than setting up a registry (in 4.9.0)

**Comment by umohnani8** [comment]
With the rework of farm build, we directly push to a registry now so  nothing is changed under the cwd. Closing this issue as it has been fixed.

@afbjorklund pushing built images to a registry was the best option we could find wrt to performance and memory to avoid moving images back and forth from various machines. The current transports we have don't support a direct copy from one location to another without creating a tmp file. If this is something you would like to discuss further please open another issue or add it as a topic to a future cabal meeting.

**Comment by afbjorklund** [comment]
I think it will work out, when there is more documentation and testing on it (probably not "GA" until v5 anyway?)
